packages/dt_sphinx_whatyouneedget/__init__.py

```diff
@@ -0,0 +1,108 @@
+"""A lightweight example directive to make it easy to demonstrate code / results."""
```

Comment:
Looks fine and tested working in books. Only feedback is that it would be nice to give it a different indicator icon to separate the appearance from the the Note admonition (maybe right and left arrow or a closed and opened box for Need and Get), but that's non-blocking.